README.md

```diff
@@ -10,9 +10,9 @@ making working with your database easier, but this comes at the cost of ineffici
 confusing code mapping. The goal of ScalaRelational is to provide a one-to-one representation of the SQL language in
 Scala and provide performance improvements to working with the database.
 
-Currently H2 and MySQL is tested, but other databases technically are supported though certain functionality may require
-some slight alterations to work properly. One of the goals of ScalaRelational is to be a straight-forward, type-safe framework
-to write SQL queries in Scala with no magic or guesswork of what the resulting SQL will be.
+Currently H2 and MariaDB (version 10.0) is tested, but other databases technically are supported though certain functionality
```

Comment:
Ok, I think we might misunderstand eachother:

- You can use MariaDB as a drop-in replacement for MySQL. And I believe the other way arround is possible.
- Some features missing from MySQL are implemented in MariaDB. So if we test against MariaDB me might miss regressions for MySQL.
- Travis has a pre-installed MySQL instance running. But can be configured (and is using this PR) to use MariaDB instead. I don't believe is easy to enable both (of course it would be writing custom install scripts).